python/ml4ir/model/optimizer.py

```diff
@@ -18,14 +22,14 @@ def get_optimizer(
     )
 
     if optimizer_key == OptimizerKey.ADAM:
-        return tf_optimizers.Adam(learning_rate=learning_rate_schedule)
+        return tf_optimizers.Adam(learning_rate=learning_rate_schedule, clipvalue=CLIP_VALUE)
```

Comment:
This helps handle gradient explosions. I've set it to a value that I found to be fairly stable from my experiments. But, we will have to fine tune this as a hyperparameter at some point.

Reply:
but should be passed in, right?  Not a global constant?

Reply:
Yes, for sure. Will do that.